nipype/algorithms/modelgen.py

```diff
@@ -414,6 +418,12 @@ def _generate_standard_design(self,
                     if hasattr(info, 'tmod') and info.tmod and \
                             len(info.tmod) > cid:
                         sessinfo[i]['cond'][cid]['tmod'] = info.tmod[cid]
+                    
+                    if hasattr(info, 'orth') and info.orth:
+                        if info.orth[cid] == 'Yes':
+                            sessinfo[i]['cond'][cid]['orth'] = 1
+                        elif info.orth[cid] == 'No':
+                            sessinfo[i]['cond'][cid]['orth'] = 0
```

Comment:
Can you rebase or revert the `orth` changes out?

Reply:
ah, silly, I checked out a new branch from another patch :|

Reply:
oops, I did a hard reset to SHA before the `orth` changes and then force-push to update the remote branch, but I didnt realize that one will close the pull request. I committed the AFNI changes again, but they dont appear here anymore.

Shall I simply do another pull request?